src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -448,6 +447,28 @@ transactionPreProcessImpl(
         return rpcError(rpcSRC_ACT_NOT_FOUND);
     }
 
+    if (signingArgs.editFields())
+    {
+        if (!tx_json.isMember(jss::Sequence))
+        {
+            bool const hasTicketSeq =
+                tx_json.isMember(sfTicketSequence.jsonName);
+            if (!hasTicketSeq && !sle)
+            {
+                JLOG(j.debug())
+                    << "transactionSign: Failed to find source account "
+                    << "in current ledger: " << toBase58(srcAddressID);
+
+                return rpcError(rpcSRC_ACT_NOT_FOUND);
```

Comment:
This block is not covered by unit tests. It is also not covered by unit tests in `develop` branch (in the old location below), so possibly it is not practical, but can you pls confirm that you looked to add the relevant test ?

Reply:
Fixed in 20abcf91ed1ccf4ab28407155d458306f37bb2de (I think - waiting for CI to finish to be sure)

Reply:
The added test did not cover those lines, digging through the code shows me that it's pretty hard to hit. Since it's not covered in `develop`, I don't think it's worth the effort of adding it here.